### ISSUE TITLE
Multi-level lists in Pandoc don't render when using officedown

### ISSUE THREAD

**Issue by rebecca-hh-rosen** [davidgohel/officedown#36]
Hello,

I'm experiencing some challenges getting multilevel lists to render with proper formatting in Word using officedown. Most notably, when I've organized a multilevel list and seek to render it using pandoc.list(), the following occurs:

<img width="591" alt="Screen Shot 2020-08-26 at 10 32 48 AM" src="https://user-images.githubusercontent.com/41338345/91317006-7fdccf00-e787-11ea-9ebb-d84a589b1937.png">

As you can see, the code renders appropriately for single level lists (like Subsection2 etc), but perverts the formatting for Section1 in both background color and bullet points.

Additionally, if I go into the created word doc and manually delete / re-enter the space between the beginning point and asterisk, a bullet automatically forms and even adjusts some of the odd formatting (ie that beige highlight behind the words).

Most curiously, as I was preparing this example, I noticed that for three-tiered lists this is not an issue. See below for an example in "Section7", where the bullets align with what is outlined in the reference document:
<img width="973" alt="Screen Shot 2020-08-26 at 10 34 20 AM" src="https://user-images.githubusercontent.com/41338345/91317169-b581b800-e787-11ea-83a9-df2bc1ffe7c0.png">

The only difference I have noted on this is that Section7 contains no trailing whitespace in the Word doc, while the others do - however, I have a call to trimws() prior to adding these points to the list, so it should not be programming it that way. Furthermore, I have tried to set pandoc.list's argument "loose" to TRUE, which only produces the same above formatting for the misaligned lists, and seems to create more of a double-spaced format for the correctly adhering lists.

Any thoughts would be helpful, as I've seen issues raised around this spacing issue in only one issue ([here](https://github.com/jgm/pandoc/issues/5645) on pandoc's github page), but found none that detail using pandoc lists for multilvel bullet points or numbered lists with any ease. I've also tried multiple reference documents, and even attempted to manually re-create pandoc.list as a function with formatting to my desired specifications, to no avail.

Thanks so much!

Please see my attached files for further reference and full code, dummy data and referenced Word document.

[officedown-REPREX-export.zip](https://github.com/davidgohel/officedown/files/5130571/officedown-REPREX-export.zip)

Results of my sessionInfo() call:
<img width="1244" alt="Screen Shot 2020-08-26 at 10 51 33 AM" src="https://user-images.githubusercontent.com/41338345/91319211-1e6a2f80-e78a-11ea-932b-79fba843ff10.png">

Note - I am working with R 4.0, but I am on my work computer and using our RStudio server, and while we recently updated, I may not have full authorization to update to some of the latest CRAN packages.

**Comment by davidgohel** [comment]
Thanks,

If I change output format from `officedown::rdocx_document` to `rmarkdown::word_document`, I get the same result. 

It seems it's an issue with `pander::pandoc.list`. For example, this is not a valid list, it's a block code (because indeted with 4 spaces):

> The md list generated after "#### Subsection5"

````
    * Talking Point Parent 
        * Talking Point Child 
        * Talking Point Child  
````

> The result :

    * Talking Point Parent 
        * Talking Point Child 
        * Talking Point Child  

To be a valid list, it should be:

> The md list after "#### Subsection5"

````
* Talking Point Parent 
  * Talking Point Child 
  * Talking Point Child  
````

> The result :

* Talking Point Parent 
  * Talking Point Child 
  * Talking Point Child

**Comment by rebecca-hh-rosen** [comment]
Ah interesting! The particular change in formatting would make sense as a code chunk. 

I will also redirect this question over to Pandoc, but do you have any insights on workaround in officedown that could help? I haven't been able to see any direct implementations of pandoc.list() in an officedown context and would be curious if there's an alternative to this process.

**Comment by rebecca-hh-rosen** [comment]
UPDATE -- I was able to fix the issue! I still have to do some more sleuthing to figure out exactly what the exact issue(s) were, but it seemed that first instantiating the nested lists as vectors (eg listy <- c() ) as opposed to lists ( eg listy <- list() ) was causing some trouble within pandoc list. Once I changed that and cleaned up the reference doc it worked fine. Thank you so much for your time and contributions!
